definitions/variable/sdg-indicators/sdg.yaml

```diff
@@ -415,3 +462,9 @@
       negative for payment)
     sdg: 17
     unit: billion USD_2010/yr
+- Unemployment:
```

Comment:
This is already proposed to be added via #57 - and in the interest of better categorization, it might be helpful to change that to *Employment|Unemployed* or similar (so that the variables for Employment and Unemployment are "close" in the variable template or the Scenario Explorer variable tree)...